src/pages/LogDrop/ParseLogSelect/index.tsx

```diff
@@ -14,13 +14,18 @@ interface ParseLogSelectProps {
   onCancel: () => void;
 }
 
+type SelectState =
+  | LogTypes.RESMOKE_LOGS
+  | LogTypes.EVERGREEN_TASK_LOGS
+  | undefined;
+
 const ParseLogSelect: React.FC<ParseLogSelectProps> = ({
   fileName,
   onCancel,
   onParse,
 }) => {
-  const [logType, setLogType] = useState<LogTypes | undefined>(
-    (Cookie.get(LAST_SELECTED_LOG_TYPE) as LogTypes) ?? undefined,
+  const [logType, setLogType] = useState<SelectState>(
+    (Cookie.get(LAST_SELECTED_LOG_TYPE) as SelectState) ?? undefined,
```

Comment:
Why did you make this change it seems to do the same thing but is arguably slightly less readable.

Reply:
`LogTypes` includes values that are unrelated to the Select component state which makes it harder to infer usage and less readable. We should be more strict with our typings and exclude unused values.